hw/ip_templates/rv_plic/fpv/tb/coverage_waivers.tcl

```diff
@@ -0,0 +1,11 @@
 dut.u_gateway.le_i[84] || dut.u_gateway.le_i[85] || dut.u_gateway.le_i[86] || dut.u_gateway.le_i[87] || dut.u_gateway.le_i[88] || dut.u_gateway.le_i[89] || dut.u_gateway.le_i[90] || dut.u_gateway.le_i[91] || dut.u_gateway.le_i[92] || dut.u_gateway.le_i[93] || dut.u_gateway.le_i[94] || dut.u_gateway.le_i[95] || dut.u_gateway.le_i[96] || dut.u_gateway.le_i[97] || dut.u_gateway.le_i[98] || dut.u_gateway.le_i[99] || dut.u_gateway.le_i[100] || dut.u_gateway.le_i[101] || dut.u_gateway.le_i[102] || dut.u_gateway.le_i[103] || dut.u_gateway.le_i[104] || dut.u_gateway.le_i[105] || dut.u_gateway.le_i[106] || dut.u_gateway.le_i[107] || dut.u_gateway.le_i[108] || dut.u_gateway.le_i[109] || dut.u_gateway.le_i[110] || dut.u_gateway.le_i[111] || dut.u_gateway.le_i[112] || dut.u_gateway.le_i[113] || dut.u_gateway.le_i[114] || dut.u_gateway.le_i[115] || dut.u_gateway.le_i[116] || dut.u_gateway.le_i[117] || dut.u_gateway.le_i[118] || dut.u_gateway.le_i[119] || dut.u_gateway.le_i[120] || dut.u_gateway.le_i[121] || dut.u_gateway.le_i[122] || dut.u_gateway.le_i[123] || dut.u_gateway.le_i[124] || dut.u_gateway.le_i[125] || dut.u_gateway.le_i[126] || dut.u_gateway.le_i[127] || dut.u_gateway.le_i[128] || dut.u_gateway.le_i[129] || dut.u_gateway.le_i[130] || dut.u_gateway.le_i[131] || dut.u_gateway.le_i[132] || dut.u_gateway.le_i[133] || dut.u_gateway.le_i[134] || dut.u_gateway.le_i[135] || dut.u_gateway.le_i[136] || dut.u_gateway.le_i[137] || dut.u_gateway.le_i[138] || dut.u_gateway.le_i[139] || dut.u_gateway.le_i[140] || dut.u_gateway.le_i[141] || dut.u_gateway.le_i[142] || dut.u_gateway.le_i[143] || dut.u_gateway.le_i[144] || dut.u_gateway.le_i[145] || dut.u_gateway.le_i[146] || dut.u_gateway.le_i[147] || dut.u_gateway.le_i[148] || dut.u_gateway.le_i[149] || dut.u_gateway.le_i[150] || dut.u_gateway.le_i[151] || dut.u_gateway.le_i[152] || dut.u_gateway.le_i[153] || dut.u_gateway.le_i[154] || dut.u_gateway.le_i[155] || dut.u_gateway.le_i[156] || dut.u_gateway.le_i[157] || dut.u_gateway.le_i[158] || dut.u_gateway.le_i[159] || dut.u_gateway.le_i[160] || dut.u_gateway.le_i[161] || dut.u_gateway.le_i[162] || dut.u_gateway.le_i[163] || dut.u_gateway.le_i[164] || dut.u_gateway.le_i[165] || dut.u_gateway.le_i[166] || dut.u_gateway.le_i[167] || dut.u_gateway.le_i[168] || dut.u_gateway.le_i[169] || dut.u_gateway.le_i[170] || dut.u_gateway.le_i[171] || dut.u_gateway.le_i[172] || dut.u_gateway.le_i[173] || dut.u_gateway.le_i[174] || dut.u_gateway.le_i[175] || dut.u_gateway.le_i[176] || dut.u_gateway.le_i[177] || dut.u_gateway.le_i[178] || dut.u_gateway.le_i[179] || dut.u_gateway.le_i[180] || dut.u_gateway.le_i[181] || dut.u_gateway.le_i[182] || dut.u_gateway.le_i[183] || dut.u_gateway.le_i[184] || dut.u_gateway.le_i[185])} -persistency {AAAAlnicPYw7CoNgEIQ/7+ABrCxSJeZXY5d7iIjxAUIKX1h6dmcVZZhl+WZ2AQ/4EpES8+bFk44fTiTRXmmvaSTL20PWcnw0rZeJdPrhM7FSMvCn103Jovz8b34wq2G8J5cLAkJ5u/l48R2lUhT+} -type {branch} -instance {dut.u_gateway} -comment {Added by GUI, apply waiver on ' 7239'} -tag {*}
```

Comment:
I'm not sure we can sensibly do this (because it's templated code and the expression depends on the template). But I think we can just get rid of the item! I've tried to do that in #26559. Does it look right to you?